goodconf/__init__.py

```diff
@@ -94,34 +127,21 @@ def customise_sources(
             file_secret_settings: SettingsSourceCallable,
         ) -> Tuple[SettingsSourceCallable, ...]:
             """Load environment variables before init"""
-            return env_settings, init_settings, file_secret_settings
-
-    def load(self, filename: str = None) -> None:
+            return (
+                init_settings,
+                env_settings,
+                file_config_settings_source,
+                file_secret_settings,
+            )
```

Comment:
Could you document the precedence here in the README?

Reply:
I just looked at the README and it already documents the precedence of env over files. I'd prefer not to document anything else currently and wait to see if pydantic 2 would require further changes. In the same sense I would not document initialization via `__init__` now. Everyone using this library and knows pydantic can use it, but let's wait and see what pydantic 2 does before making this an official feature.